packages/devtools_app/lib/src/flutter/common_widgets.dart

```diff
@@ -154,3 +156,92 @@ ErrorReporter showErrorSnackBar(BuildContext context) {
     );
   };
 }
+
+Widget clearButton({
```

Comment:
Suggestion: make these StatelessWidgets, and add some documentation about how they're supposed to be used in different contexts.

Reply:
done